modules/spring-config/src/main/resources/META-INF/mule-module.xsd

```diff
@@ -219,4 +224,20 @@
             <xsd:enumeration value="PRIMARY"/>
         </xsd:restriction>
     </xsd:simpleType>
+
+    <xsd:simpleType name="useType">
+        <xsd:annotation>
+            <xsd:documentation>
+                Set of defined uses for a given property/parameter.
+                REQUIRED marks the attribute as required in the XSD, failing if leaved empty when consuming the parameter/property. It can not be REQUIRED if the parameter/property has a defaultValue;
```

Comment:
I would remove the XSD part of the description. This is meant to be used within Studio as inline doc. when you are using the attribute in the UI. I don't think that it matters to the user that it will be required in the XSD. I think they would like to just understand the meaning of what's going to happen when working with the operation they created. What do you think?.

Same with OPTIONAL